File: ios/GitaVani/GitaVani/Models/Chapter.swift
import Foundation

struct Chapter: Codable, Identifiable {
    let chapterNumber: Int
    let versesCount: Int
    let name: String
    let translation: String
    let transliteration: String
    let meaning: LocalizedText
    let summary: LocalizedText

    var id: Int { chapterNumber }
}

File: android/GitaVani/app/src/main/java/com/nikhilsi/gitavani/model/Chapter.kt
package com.nikhilsi.gitavani.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Chapter(
    @SerialName("chapter_number") val chapterNumber: Int,
    @SerialName("verses_count") val versesCount: Int,
    val name: String,
    val translation: String,
    val transliteration: String,
    val meaning: LocalizedText,
    val summary: LocalizedText
)
